src/Backend/Core/Engine/Authentication.php

```diff
@@ -192,7 +192,7 @@ public static function isAllowedAction($action = null, $module = null)
     {
         // always allowed actions (yep, hardcoded, because we don't want other people to fuck up)
         $alwaysAllowed = array(
-            'Dashboard' => array('Index' => 7),
+            //'Dashboard' => array('Index' => 7),
```

Comment:
I don't even think this should be changed. I think only the changes in Authentication::processRegularRequest are needed to fix the issue.